env/environment.go

```diff
@@ -8,36 +8,34 @@ import (
 
 // Environment is a map of environment variables, with the keys normalized
 // for case-insensitive operating systems
```

Comment:
I imagine the “with the keys normalized” part might be the original reason for wrapping the map?
To guard against non-normalized reads/writes.
But as you say, the liberal passing of `.ToMap()`'s mutable result to other scopes renders that less useful.

Reply:
my kingdom for mutabilty modifiers... time to rewrite in rust?